0.download_data/README.md

```diff
@@ -0,0 +1,20 @@
+# Download JUMP-Target SQLite files from AWS
+
+In this module, we download the SQLite files from [AWS](https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/).
+There are 51 plates from the pilot dataset, totalling 1.1 TB of storage from the SQLite files.
+
+Firstly, we generate a manifest file in the [data folder](./data/) called [jump_dataset_location_manifest.csv](./data/jump_dataset_location_manifest.csv).
+Then, we use the [download_from_aws.sh](./download_from_aws.sh) file, which contains the bash script that will download the files from the paths in the manifest.
+We call this file in a notebook to make it fully reproducible.
```

Comment:
I'm not sure what this sentence adds? Can you either clarify or remove?

Reply:
I can remove it. I included it since I haven't seen before an `sh` file used in a notebook before to perform a function that wasn't to run python scripts and nbconvert so I was clarifying for myself.